src/lib/utils/time.test.ts

```diff
@@ -0,0 +1,22 @@
+import { describe, expect, it } from 'vitest'
+import { isLastYear } from './time'
+
+const currentDate = new Date()
+const dateFromLastYear = new Date(
+  currentDate.getFullYear() - 1,
+  currentDate.getMonth(),
+  currentDate.getDate()
+)
+
+//Chat with date from last year
```

Comment:
This comment doesn't really make anything clearer - the code is super-clear in itself and doesn't need explaining. In general, comments shouldn't be used to say what the code does, they should explain *why* the code does what it does.

Reply:
Nice tests btw 👍